_aicspylibczi/TypedImage.h

```diff
@@ -96,22 +97,27 @@ namespace pylibczi {
       return this->operator[](calculateIdx(zeroPadded));
   }
   template<typename T>
-  inline void TypedImage<T>::loadImage(const std::shared_ptr<libCZI::IBitmapData>& bitmap_ptr_, size_t channels_)
+  inline void TypedImage<T>::loadImage(const std::shared_ptr<libCZI::IBitmapData>& bitmap_ptr_, libCZI::IntSize size_,
+      size_t channels_)
   {
-      libCZI::IntSize size = bitmap_ptr_->GetSize();
+      // libCZI::IntSize size = bitmap_ptr_->GetSize();
```

Comment:
looks like you can remove this and the extra curly brackets below it

Reply:
Thanks it was a leftover from messing with threadlocking.